app/src/main/res/values/strings.xml

```diff
@@ -91,6 +92,8 @@ Tap this message (or hit back) to skip this step.</string>
   <string name="use_previous">Use previous title/description</string>
   <string name="allow_gps">Automatically get current location</string>
   <string name="allow_gps_summary">Retrieve current location to offer category suggestions if image is not geotagged</string>
+  <string name="preference_theme">Select theme</string>
```

Comment:
Maybe a strings would be:
preference_theme: "Night mode"
preference_theme_summary: "Use dark theme"